dimod/core/initialized.py

```diff
@@ -19,16 +19,21 @@
 this abstract base class in a sampler.
 
 """
+from __future__ import annotations
 
 import abc
 
 from collections import namedtuple
 from numbers import Integral
 
+from typing import Optional, Tuple
```

Comment:
Ok, I know I do this everywhere so it's very hypocritical of me, but recently I've starting just `import typing` and doing `typing.Tuple` etc everywhere. In some files like `binary_quadratic_model.py` it really makes sense to import at least `Union`, `Tuple`, and `Optional`, but in a file like this I'd rather keep the namespace clean if that makes sense. This is still something that I am trying to build best practices around though so forgive me if I change around.